tests/components/recorder/test_purge.py

```diff
@@ -161,19 +165,17 @@ def test_purge_method(self):
             # now we should only have 2 events left
             assert events.count() == 2
 
-            assert not (
-                "EVENT_TEST_PURGE" in (event.event_type for event in events.all())
-            )
+            assert not ("EVENT_TEST_PURGE" in (event.event_type
+                                               for event in events.all()))
 
             # run purge method - correct service data, with repack
-            with patch(
-                "homeassistant.components.recorder.purge._LOGGER"
-            ) as mock_logger:
+            with patch("homeassistant.components.recorder.purge._LOGGER"
+                       ) as mock_logger:
                 service_data["repack"] = True
-                self.hass.services.call("recorder", "purge", service_data=service_data)
+                self.hass.services.call("recorder",
+                                        "purge",
+                                        service_data=service_data)
                 self.hass.block_till_done()
                 self.hass.data[DATA_INSTANCE].block_till_done()
-                assert (
-                    mock_logger.debug.mock_calls[3][1][0]
-                    == "Vacuuming SQL DB to free space"
-                )
+                assert (mock_logger.debug.mock_calls[3][1][0] ==
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)